Manito/Manito/Screens/Main/MainViewController.swift

```diff
@@ -294,15 +257,15 @@ final class MainViewController: BaseViewController {
         ParticipateRoomVC.modalPresentationStyle = .fullScreen
         ParticipateRoomVC.modalTransitionStyle = .crossDissolve
 
-        present(ParticipateRoomVC, animated: true, completion: nil)
+        self.present(ParticipateRoomVC, animated: true, completion: nil)
```

Comment:
p3;

요기도 지워도 될 것 같아요 !

```suggestion
        self.present(ParticipateRoomVC, animated: true)
```

Reply:
[5b78e36](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/375/commits/5b78e369fc833d0907b44a105c6953832ed757cf)
반영했습니다.